tensorflow_datasets/scripts/random_img_generator.py

```diff
@@ -0,0 +1,102 @@
+import os
+from zipfile import ZipFile
+from PIL import Image
+import numpy as np
+import tarfile
+import zipfile
+import shutil
+import gzip
+
+
+# images_path variable to store the path of images for later compression
+images_path = []
+
+
+def image_process(root):
+    """ Generate random images and remove noise of the
+        images to compress better.
+
+          Args:
```

Comment:
Please take a note of indentation in descriptions. You should change that as follows:
```py
""" 
Generate random images and remove noise of the
images to compress better.

Args:
  root: path of the images to get processed
"""
```
Also, update the indentation in other methods too in the same way.

Reply:
Thanks you ashutosh i will make the changes in indentations and this code is pylint resolved. I have already run pylint on this code.